bitnami/nginx/templates/deployment.yaml

```diff
@@ -39,19 +39,9 @@ spec:
         - name: http
           containerPort: 8080
         livenessProbe:
```

Comment:
You can do it on a single line:

```diff
-         livenessProbe:
- {{ toYaml .Values.livenessProbe | indent 10 }}
+         livenessProbe: {{ toYaml .Values.livenessProbe | mindent 10 }}
```

Reply:
Good idea, looks like it should actually be `nindent` though. I'll go ahead and make this change.

Reply:
Yes, sorry for the typo